hack/ci/e2e-k8s.sh

```diff
@@ -106,15 +113,14 @@ create_cluster() {
       \"logging-format\": \"${KUBELET_LOG_FORMAT}\""
   fi
 
-  # JSON map injected into featureGates config
-  feature_gates="{}"
-  # --runtime-config argument value passed to the API server
-  runtime_config="{}"
+  # JSON or YAML map injected into featureGates config
+  feature_gates="${FEATURE_GATES:-{\}}"
+  # --runtime-config argument value passed to the API server, again as a map
+  runtime_config="${RUNTIME_CONFIG:-{\}}"
 
```

Comment:
A [prior version of this PR](https://github.com/kubernetes-sigs/kind/compare/9bde01149e6be33aad81dd1e57dd7d9fe7db413d..f5ece4804021e04dd3d4352b23dbec2ec296440a) allowed both to be set and used shell code to parse and merge all settings. This was deemed too complex and now the options are mutually exclusive: setting `GA_ONLY` overrides `FEATURE_GATES/RUNTIME_CONFIG`.

Reply:
FWIW, I think that's sufficient: someone who has a need to set features individually can also add the `"api/alpha":"false", "api/beta":"false"` there. The special case for `"certificates.k8s.io/v1beta1":"true"` seems irrelevant now.

Reply:
in the GA_ONLY=true case below, can we detect and error if $feature_gates or $runtime_config are anything other than `{}`, rather than silently ignoring the envvar-set values

Reply:
Added.